src/test/java/seedu/address/model/internship/predicate/StatusIsWishlistPredicateTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.address.model.internship.predicate;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.status.Status;
+import seedu.address.testutil.InternshipApplicationBuilder;
+
+class StatusIsWishlistPredicateTest {
+    @Test
+    public void equals() {
+        StatusIsInterviewPredicate predicate = new StatusIsInterviewPredicate();
+
+        // same object -> returns true
+        assertTrue(predicate.equals(predicate));
+
+        // different types -> returns false
+        assertFalse(predicate.equals(1));
+
+        // null -> returns false
+        assertFalse(predicate.equals(null));
+    }
+
+    @Test
+    public void test_statusIsInterview_returnsTrue() {
+        StatusIsWishlistPredicate predicate = new StatusIsWishlistPredicate();
+
+        assertTrue(predicate.test(new InternshipApplicationBuilder().withStatus(Status.WISHLIST).build()));
+    }
+
+    @Test
+    public void test_statusIsNotInterview_returnsFalse() {
```

Comment:
I think interview needs to be changed to wishlist for method names

Reply:
This as well!